### PR TITLE
2179 concent/forced download transfer rate failure

### LINKED ISSUE #2179: [Concent] UC2 / Forced Download - Transfer Rate failure
Support for the Transfer Rate Failure reason to request Concent assistance in the "Forced Download" use case (use case 2)

### PR DESCRIPTION
**Pull request by shadeofblue**
closes #2179 

+ add `size` to `ReportComputedTask`

### CODE REVIEW

File: tests/factories/messages.py [hunk 1]
```diff
@@ -83,7 +84,8 @@ class Meta:
         TaskToCompute,
         compute_task_def__subtask_id=factory.SelfAttribute('...subtask_id'),
     )
-    size = factory.Faker('pyint')
+    size = factory.LazyFunction(
+        lambda: random.randint(1 << 20, 10 << 20))
```
Comment (jiivan) [inline tests/factories/messages.py]: `factory.Faker('random_int', min=1 << 20, max=10 << 20)`?

File: tests/factories/taskserver.py [hunk 2]
```diff
@@ -32,3 +32,5 @@ class Meta:
     owner_key_id = factory.Faker('sha1')
     owner = factory.SubFactory('tests.factories.p2p.Node')
     package_sha1 = factory.Faker('sha1')
+    result_size = factory.LazyFunction(
+        lambda: random.randint(1 << 20, 10 << 20))
```
Comment (jiivan) [inline tests/factories/taskserver.py]: `factory.Faker('random_int', min=1 << 20, max=10 << 20)`?

**Review by jiivan — APPROVED** [review]
LGTM. Small comment about random providers from faker.
https://github.com/joke2k/faker/blob/07ca4ede54c26554fdb5c7a4f55432cb0498d338/faker/providers/__init__.py

File: tests/factories/taskserver.py (hunk 2 above)
Comment (shadeofblue) [reply tests/factories/taskserver.py]: :+1:

File: tests/factories/messages.py (hunk 1 above)
Comment (shadeofblue) [reply tests/factories/messages.py]: :+1:

**Pull request merged**